README.md

```diff
@@ -19,45 +22,67 @@ You can use any preferred technology to create Saleor Apps, but Next.js is among
 ## How to use this project
 
 ### Saleor CLI for the win 🚀
+
 [Saleor CLI](https://github.com/saleor/saleor-cli) is designed to save you from the repetitive chores around Saleor development, including creating Apps. It will take the burden of spawning new apps locally, connecting them with Saleor environments, and establishing a tunnel for local development in seconds.
 
 [Full Saleor CLI reference](https://docs.saleor.io/docs/3.x/developer/cli)
 
 If you don't have (free developer) Saleor Cloud account create one with the following command:
+
 ```
 saleor register
 ```
 
 Now you're ready to create your first App:
+
 ```
 saleor app create [your-app-name]
 ```
 
 In this step, Saleor CLI will:
+
 - clone this repository to the specified folder
 - install dependencies
 - ask you whether you'd like to install the app in the selected Saleor environment
 - create `.env` file
 - start the app in development mode
 
 Having your app ready, the final thing you want to establish is a tunnel with your Saleor environment. Go to your app's directory first and run:
+
 ```
 saleor app tunnel
 ```
+
 Your local application should be available now to the outside world (Saleor instance) for accepting all the events via webhooks.
 
 A quick note: the next time you come back to your project, it is enough to launch your app in a standard way (and then launch your tunnel as described earlier):
 
 ```
 pnpm dev
 ```
-or 
+
+or
+
 ```
 npm run dev
 ```
 
-### Manual installation
-If you don't want to use our CLI tool, just clone this repository and use standard tools instead. Learn how to [install your app manually](https://docs.saleor.io/docs/3.x/developer/extending/apps/installing-apps#installation-using-graphql-api) at any Saleor instance from our general docs and use tunneling tools like [localtunnel](https://github.com/localtunnel/localtunnel) or [ngrok](https://ngrok.com/) in order to expose it to the external world if needed.
+### Local development without CLI
+
+Add URL of your Saleor instance to the `.env` file:
+
+```
+NEXT_PUBLIC_SALEOR_HOST_URL=https://your-saleor-instance.com
+```
+
+Install dependencies `pnpm install`
+
+Start local server `npm run dev`
+
+Follow the guide [how install your app](https://docs.saleor.io/docs/3.x/developer/extending/apps/installing-apps#installation-using-graphql-api) and use tunneling tools like [localtunnel](https://github.com/localtunnel/localtunnel) or [ngrok](https://ngrok.com/) in order to expose your local server.
+
+### Generated schema and typings
 
+Commands `build` and `dev` would generate schema and typed functions using Saleor's GraphQL endpoint. Commit `generated` folder to your repo as they are necessary for queries and keeping track of the schema changes.
```

Comment:
@magul you would need to commit at least ```graphql.ts``` as these are actual TS hooks for fetching data. The build would fail without this file.

Reply:
A short summary we decided that so far:
1. users track their schema on their own
2. we should **NOT** keep TypeScript types in the repo as it's not a good practice (`codegen` creators advise against it) and it generates unnecessary noise 
3. `build`, `start` and `dev` already have (pre-) hooks so that those types will be always generated in a determinisitic manner as pointed by @magul 

Also, I propose (for simplicity) to keep only SDL schema: less moving parts, easier support in the long term